frontend/src/components/RepositoryVCSProviderPanel.vue

```diff
@@ -38,12 +38,11 @@ export default { name: "RepositoryVCSProviderPanel" };
 </script>
 
 <script setup lang="ts">
-import { useStore } from "vuex";
 import { reactive, computed, watchEffect, onUnmounted, onMounted } from "vue";
 import isEmpty from "lodash-es/isEmpty";
 import { OAuthWindowEventPayload, openWindowForOAuth, VCS } from "../types";
 import { isOwner } from "../utils";
-import { pushNotification, useCurrentUser } from "@/store";
+import { pushNotification, useCurrentUser, useVcsStore } from "@/store";
```

Comment:
We should name it as `useVCSStore` not `useVcsStore`, just like `vcsStore.createVCS`.